dashboard/generate/definitions.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+
+
+TEMPLATES_PATH = Path(__file__).parents[1] / "templates"
+PULL_REQUESTS_TEMPLATE = "pull_requests.html.template"
+CONSTANTS_TEMPLATE = "constants.tsx.template"
```

Comment:
Added a definitions file to help with the shared variables in the package

Reply:
Allright! This LGTM, eventually we want to use `duckdb-wasm`